config/routing.go

```diff
@@ -106,6 +108,7 @@ type RouterType string
 const (
 	RouterTypeHTTP       RouterType = "http"       // HTTP JSON API for delegated routing systems (IPIP-337).
 	RouterTypeDHT        RouterType = "dht"        // DHT router.
+	RouterTypeDHTv2      RouterType = "dhtv2"      // DHT router using go-libp2p-kad-dht/v2.
```

Comment:
I don't think this name is correct, this router is compatible with v1.
So I don't think v2 applies here.

Reply:
It will allow different custom parameters, which I haven't added yet.

Reply:
I mean network wise, it's just a different implementation, it's it will talk with other v1 dht servers and clients.
I would name it `go-kademlia`

Reply:
I see how this name could cause confusion between go-libp2p-kad-dht version 2 and version 2 of the dht protocols. I'll choose a better name - same for boxo

Reply:
Renamed router type to `dht/zikade`